support_scripts/audit_objects.rb

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+require File.expand_path(File.dirname(__FILE__) + "/../config/environment")
+
+require 'avalon/audit_objects'
+
+Avalon::AuditObjects.run
```

Comment:
Would make these support_scripts as rake tasks under `lib/tasks`? Could have a `support_script.rb` task and namespaced accordingly, then you can just do `Rake -T`  and see all the tasks? Less reinventing the wheel and more rails standard?